package.json

```diff
@@ -35,21 +36,22 @@
   "devDependencies": {
     "@babel/runtime": "^7.8.4",
     "@mendix/native-mobile-toolkit": "latest",
-    "detox": "17.14.5",
+    "detox": "19.13.0",
```

Comment:
Why are we keeping detox here?

Reply:
Because detox is only available in NT not in MiN anymore

Reply:
And why do we keep detox there while we don't use it anymore?

Reply:
Check the code! Detox still lives in native template and @ablok is actually trying to execute the automation over it.